### ISSUE TITLE
ACRN debian package not complete when source is not cloned to standard folder

### ISSUE THREAD

**Issue by shiqingg** [projectacrn/acrn-hypervisor#7022]
When cloning the ACRN source code to a folder, which is not named "acrn-hypervisor", the debian package created by the "make" call does not contain all needed files.
This is most likely due to the fact, that the file ".deb.conf" contains hardcoded folder names which contain the folder "acrn-hypervisor". Nevertheless, the build does not fails, which it should...

**Comment by shiqingg** [comment]
[External_System_ID] ACRN-7584

**Comment by NanlinXie** [comment]
@shiqingg , could you confirm whether this issue is resolved by merged patch?

**Comment by shiqingg** [comment]
> @shiqingg , could you confirm whether this issue is resolved by merged patch?

@NanlinXie Yes, it can be solved by this patch. PR#7016 for master branch. PR#7023 for release_2.7 branch.

**Comment by NanlinXie** [comment]
Closed this issue with verified result.
